stdlib/3/lib2to3/pgen2/driver.pyi

```diff
@@ -0,0 +1,24 @@
+# Stubs for lib2to3.pgen2.driver (Python 3.6)
+
+from typing import Any, Callable, IO, Iterable, List, Optional, Tuple
+
+from logging import Logger
+from lib2to3.pytree import _Context, _NL
+from lib2to3.pgen2.grammar import Grammar
+
+_RawNode = Tuple[int, str, _Context, List[_NL]]
+_Convert = Callable[[Grammar, _RawNode], _NL]
+
+
+class Driver:
+    grammar: Grammar
+    logger: Logger
+    convert: _Convert
+    def __init__(self, grammar: Grammar, convert: Optional[_Convert] = ..., logger: Optional[Any] = ...) -> None: ...
```

Comment:
logger is an `Optional[Logger]` (or at least you should be consistent with `load_grammar` below, which also takes a logger).